Microsoft.Toolkit.Uwp/Helpers/ObjectStorage/BaseObjectStorageHelper.cs

```diff
@@ -105,9 +105,9 @@ public void Save<T>(string key, T value)
         /// <param name="filePath">Path to the file that will contain the object</param>
         /// <param name="value">Object to save</param>
         /// <returns>Waiting task until completion</returns>
```

Comment:
What do you think about `Returns the <see cref="StorageFile"/> that the content is saved to.`?